Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ModelChoiceList class is deprecated since SF 2.7'.
```

Comment:
Certainly, but this is not the issue. As far you deprecate any class on the sonata code, this means it would be removed on next major.